milabench/remote.py

```diff
@@ -183,9 +278,9 @@ def milabench_remote_install(pack, setup_for="worker") -> SequenceCommand:
         return VoidCommand(pack)
 
     argv = sys.argv[2:]
-
     return SequenceCommand(
         milabench_remote_setup_plan(pack, setup_for),
+        milabench_remote_command(pack, "pin", *argv, run_for=setup_for),
```

Comment:
I think pin should be done on a different job, we might not want to update the dependencies all the time

Reply:
Will remove